Breeze.UI/src/app/wallet/status-bar/status-bar.component.ts

```diff
@@ -27,7 +26,7 @@ export class StatusBarComponent implements OnInit {
   constructor(private apiService: ApiService, private globalService: GlobalService, private genericModalService: ModalService) { }
 
   ngOnInit() {
-    this.startSubscriptions();
+    this.startSubscriptions() ;
```

Comment:
Why space?

Reply:
Not something I touched.  Have changed all the same.